### PR TITLE
Fixes #30

### LINKED ISSUE #30: Remove csrfToken in PrgComponent
Do you think it would make sense to remove the `_csrfToken` in the `PrgComponent`? In general it's not necessary but it would keep the URL clean. What do you think? If you agree I will create a PR.

### PR DESCRIPTION
**Pull request by burzum**

### CODE REVIEW

**Comment by steefaan** [comment]
:+1:

File: src/Controller/Component/PrgComponent.php [hunk 1]
```diff
@@ -7,10 +7,22 @@
 class PrgComponent extends Component
 {
     /**
+     * $_defaultConfig For the Component.
+     *
+     * ### Options
+     * - `stripParams` Fields to strip from the data to query param conversion. Strips the `__csrfToken` by default.
```
Comment (ADmad) [inline src/Controller/Component/PrgComponent.php]: `__csrfToken` => `_csrfToken`

**Comment by ADmad** [comment]
Ideally Cake itself should strip `_csrfToken` from `request->data`.

**Comment by burzum** [comment]
@ADmad any reason this is still open? :(

**Comment by ADmad** [comment]
cakephp/cakephp#7525 fixes the issue with csrf token. So is the change really necessary? I don't see the need to remove any other field to be common enough use case.

**Comment by burzum** [comment]
OK, but then the minimum required CakePHP version of the search plugin should be raised to whatever version this fix is going to be in.

**Comment by ADmad** [comment]
Yeah we can bump cake version as well as plugin's minor version after #40 is merged.

**Pull request closed without merging**